### PR TITLE
Add Blahker by @ethanhuang13

### PR DESCRIPTION
**Pull request by ethanhuang13**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -10488,6 +10488,24 @@
...
+      "category-ids": [
+        "content-blocking"
+      ],
+      "description": "Safari content blocker for interstitial ads, mainly for websites in Taiwan",
+      "license": "mit",
+      "source": "https://github.com/ethanhuang13/blahker",
+      "itunes": "https://itunes.apple.com/tw/app/blahker-ba-la-ke-gai-ban-guang/id1182699267",
+      "date_added": "Apr 2 2018 +0800",
+      "suggested_by": "@ethanhuang13",
+      "stars": 4.9,
```
Comment (dkhamsing) [inline contents.json]: `stars` would be the number of stars on GitHub, so 70 at the moment

File: contents.json [hunk 2]
```diff
@@ -10488,6 +10488,24 @@
...
+    },
+        {
+      "title": "Blahker",
+      "category-ids": [
+        "content-blocking"
+      ],
+      "description": "Safari content blocker for interstitial ads, mainly for websites in Taiwan",
+      "license": "mit",
+      "source": "https://github.com/ethanhuang13/blahker",
+      "itunes": "https://itunes.apple.com/tw/app/blahker-ba-la-ke-gai-ban-guang/id1182699267",
```
Comment (dkhamsing) [inline contents.json]: Can we remove `/tw`

**Comment by dkhamsing** [comment]
Can we also add `"lang": "zho"`?

**Comment by dkhamsing** [comment]
Thank you! Pls make the changes suggested if possible 🙏

File: contents.json (hunk 2 above)
Comment (ethanhuang13) [reply contents.json]: Yes. Just want to point out it's mainly for websites in Taiwan.

File: contents.json (hunk 1 above)
Comment (ethanhuang13) [reply contents.json]: Oh, OK~

**Comment by ethanhuang13** [comment]
> Can we also add "lang": "zho"?
OK

**Comment by ethanhuang13** [comment]
Change made. Please proceed.

**Comment by dkhamsing** [comment]
Thanks!

**Comment by ethanhuang13** [comment]
Hello, 

I'm not sure why, but the flag should be 🇹🇼 instead of 🇨🇳.

**Comment by dkhamsing** [comment]
hm sorry the flag represents the language.. is that still accurate ?

**Comment by ethanhuang13** [comment]
c is the flag of China, which use Simplified Chinese.
Blahker's UI use Traditional Chinese, which is primarily used in Taiwan🇹🇼.
Both are Chinese in speech, but different in writing/UI.

ISO 639-2 use **zho** for both Chinese.
iOS use **zh-Hans** and **zh-Hant** to distinguish them.

I'm not sure which language code or definition you are using.
Just want to point out **zho** is not enough to distinguish Simplified or Traditional Chinese.

**Comment by scribblemaniac** [comment]
Currently we use ISO 639-2. Simplified Chinese and Traditional Chinese are different character sets not different languages as you say, which is why there is no different ISO codes for them.

From my brief research of languages in Taiwan ([Source](https://en.wikipedia.org/wiki/Languages_of_Taiwan)), it looks like there are two main languages in Taiwan: Mandarin and Taiwanese Hokkien. Both of these happen to fall under the zho macrogroup, but can slightly better specified with the ISO 639-3 codes CMN and NAN respectively, but those should both also probably be represented with a Chinese flag because they encompass all dialects, not just the Taiwanese ones.

This isn't the first time an issue like this has happened, so maybe we could use an IETF language tag to add an optional dialect suffix? That would however mean more cases for the README generator to handle, and a less strict schema since they can be in varying formats. @dkhamsing @vpeschenkov Thoughts?

**Comment by dkhamsing** [comment]
While I appreciate the specificity, I believe we’re splitting hair here for the purpose of this list - Ethan I hope this is not too upsetting for us to leave your listing unchanged. I think we should of course reconsider if more of these issues crop up 

Thank you @scribblemaniac for your response 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ethanhuang13** [comment]
I suggest we open a issue to keep track of this. 

I also think we can use ISO 639-1 (two-letter for region/country) plus ISO 15924 (plus four letter for script variations), just like [iOS](https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html) uses **zh-Hans** and **zh-Hant** for this case.

**Comment by shunchu** [comment]
Great list!

While I understand the purpose of the flags is to identify non-English projects, I actually mentally associated the flags to the origin of each project. So having a PROC flag on a project originated from Taiwan or a Spanish flag on a project from Mexico actually introduces negative mental affordance. I'd suggest to just remove the visual noise/associations. It's perfectly fine to just label them with text like savages!

**Comment by dkhamsing** [comment]
@shunchu you may be onto something - thx everyone for your input - will create an issue to brainstorm 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**